Sources/HaCTML/Attributes.swift

```diff
@@ -0,0 +1,32 @@
+private func stringAttribute(_ key: String) -> AttributeKey<String> {
+  return .init(key, render: { .text($0) })
+}
+
+private func numberAttribute(_ key: String) -> AttributeKey<Int> {
+  return .init(key, render: { .text(String($0)) })
+}
+
+private func CSSAttribute(_ key: String) -> AttributeKey<[String: String]> {
+  return .init(key, render: { .text($0.map({ "\($0): \($1);" }).joined(separator: " ")) })
+}
+
+public struct Attributes {
+  // We are filling this list in incrementally, implementing attributes as we need them so that we can type them correctly
+  static public let alt = stringAttribute("alt")
+  static public let charset = stringAttribute("charset")
+  static public let className = stringAttribute("class")
```

Comment:
This is true, but this will be setting precedent. It may be the case that we'll create components that can take their own `class` as a parameter. The backtick pattern will enter userland.

There is rationale for using `className`, it is how you get that attribute in JS, and React inherited that idiom as well. 

This one might require user testing.